app/templates/play/campaign-view.pug

```diff
@@ -130,6 +134,16 @@ if showGameDevAlert
             else if level.unlocksHero
               img.treasure-chest(src="/images/pages/play/gold-chest.png")
         div(style="left: " + level.position.x + "%; bottom: " + level.position.y + "%",class="level-shadow" + (level.next ? " next" : "") + (level.locked ? " locked" : "") + " " + (levelStatusMap[level.slug] || ""))
+        if view.shouldShow('blazing-battle')
```

Comment:
Let's name this more flexibly, since the actual arena will change 3-6 times a year. Maybe `shouldShow('league-arena')`?

Reply:
Excellent feedback thank you!